test/src/module/command/restoreTest.c

```diff
@@ -1337,32 +1352,6 @@ testRun(void)
 
         TEST_RESULT_LOG("P00 DETAIL: databases found for selective restore (1, 12168, 16380, 16381, 16384, 16385)");
 
-        // -------------------------------------------------------------------------------------------------------------------------
-        TEST_TITLE("error on system database with non-systemId selected");
-
-        argList = strLstDup(argListClean);
-        strLstAddZ(argList, "--db-include=16385");
-        harnessCfgLoad(cfgCmdRestore, argList);
-
-        TEST_ERROR(
-            restoreSelectiveExpression(manifest), DbInvalidError,
-            "system databases (template0, postgres, etc.) are included by default");
-
-        TEST_RESULT_LOG("P00 DETAIL: databases found for selective restore (1, 12168, 16380, 16381, 16384, 16385)");
-
-        // -------------------------------------------------------------------------------------------------------------------------
-        TEST_TITLE("error on system database with non-systemId selected, by name");
```

Comment:
Why was this test removed? Can you point me to where we are testing that --db-include when given a system database name with a non-system-id is bring tested?

Reply:
Same as above. I'll add those 2 back.

Reply:
fixed in commit a69c81b